Jenkinsfile.treecompose

```diff
@@ -42,15 +40,13 @@ node(NODE) {
         """ }
 
         stage("Pull and run oscontainer") { sh """
-            rm ${treecompose_workdir} -rf
-            mkdir -p ${treecompose_workdir}
             rm -rf /var/lib/containers && mkdir /var/lib/containers
-            mkdir -p ${treecompose_workdir}/containers
-            mount --bind ${treecompose_workdir}/containers /var/lib/containers
+            mkdir -p ${host_workspace}/containers
+            mount --bind ${host_workspace}/containers /var/lib/containers
             skopeo inspect docker://${OSCONTAINER_IMG}
             podman pull ${OSCONTAINER_IMG}
             cid=\$(podman run --net=host -d --name oscontainer --entrypoint sleep ${OSCONTAINER_IMG} infinity)
-            ln -sf \$(podman mount \${cid})/srv/repo ${treecompose_workdir}/repo
+            ln -sf \$(podman mount \${cid})/srv/repo ${host_workspace}/repo
```

Comment:
`repo` is defined on line 33 as `${host_workspace}/repo`...is there a reason for not using `${repo}` everywhere there is `${host_workspace}/repo`?

Reply:
Yeah that'd probably be saner but...hold, I am testing another PR that's simpler.